hilti/toolchain/src/compiler/optimizer.cc

```diff
@@ -174,6 +185,7 @@ struct FunctionVisitor : OptimizerVisitor {
 
         switch ( _stage ) {
             case Stage::COLLECT: {
+                // std::cerr << "NOPE COLLECT FIELD " << n->canonicalID() << '\n';
```

Comment:
stray? also below.

Reply:
Thanks, I missed them; they are debugging statements which shouldn't be committed (I used a identifiable prefix to catch them more easily but alas 🤷)